src/scrape_up/github/pull_request.py

```diff
@@ -13,6 +13,15 @@ def __scrape_page(self):
         data = requests.get(f"https://github.com/{self.username}/{self.repository}/pull/{self.pr_number}")
         data = BeautifulSoup(data.text,"html.parser")
         return data
+    def labels(self):
+        labels_found=[]
```

Comment:
Include this method inn `documentation.md` file @madmaverickminion

Reply:
Done!